### DISCUSSION TITLE
Request for a volume control shortcut

### DISCUSSION THREAD

**Discussion by xingyunjohn1** [File-New-Project/EarTrumpet#1024] [Ideas]
Is it posssible to control main volume with scroll wheel while the cursor hovered over the EarTrumpet icon in the task bar? I suppose it will be useful.

Task bar actions May be...

- hovered: scroll to adjust
- left click: open SndVol
- right click: menu
- central (scroll) click: mute all (main)

Thanks!

**Comment by riverar** [comment]
This is already supported!

**Reply by xingyunjohn1** [reply]
Oooooh, great!

But when the cursor hovered over the EarTrumpet icon in the task bar, there is also a prompting message (windows function). The percentage of volume in the message does not change when scrolling, although the volume has been adjusted in fact.

*And this problem also appears in taskmgr's prompting message, which means may be windows prompting message does not support such function.*

Anyway, it would be much better to update and show volume percentage when scrolling. Or it is hard to find it out by only 4 states of the task bar icon. This may be  implemented by many ways, eg.
- Try to refresh the windows prompting message if possible.
- A third party floating window lies above the task bar icon or on the center of the screen.
- Show the windows 10 volume bar on the left top of screen. This bar looks like:
![Pic from the Internet](https://img-blog.csdnimg.cn/img_convert/c4e4b3c09e131294f8df5b5168a65c4e.png)

Thanks for your great job and great ideas!
